GA-worksheet.md

```diff
@@ -25,15 +20,15 @@
 
 ##Project #2 Part #2: Style the Header with CSS
 
-1. What advantages does the `<link>` tag have over the `<style>` tag?
+1. What advantages does the `<link>` tag have over the `<style>` tag?A link between a document and an external resource but the style just changes the color
```

Comment:
The `<style>` tags are a way to add many styles directly to the HTML document--not just color. The `link` tag does connect an external resource, but could you tell me what the advantage is?